plugins/modules/zabbix_usergroup.py

```diff
@@ -0,0 +1,597 @@
+            - On C(absent) will remove a user group if it exists.
+        required: false
+        type: str
+        default: "present"
+        choices: [ "present", "absent" ]
+notes:
+    - Only Zabbix >= 4.0 is supported.
+extends_documentation_fragment:
+- community.zabbix.zabbix
+'''
+
+EXAMPLES = r'''
+# Base create user group example
+- name: Create user group
+  community.zabbix.zabbix_usergroup:
+    server_url: "http://zabbix.example.com/zabbix/"
+    login_user: admin
+    login_password: secret
+    name: ACME
+    state: present
+
+# Base create user group with disabled gui access
+- name: Create user group with disabled gui access
+  community.zabbix.zabbix_usergroup:
+    server_url: "http://zabbix.example.com/zabbix/"
+    login_user: admin
+    login_password: secret
+    name: ACME
+    gui_access: disable
+
+# Base create user group with permissions
+- name: Create user group with permissions
+  community.zabbix.zabbix_usergroup:
+    server_url: "http://zabbix.example.com/zabbix/"
+    login_user: admin
+    login_password: secret
+    name: ACME
+    rights:
+        - host_group: Webserver
+          permission: read-write
+        - host_group: Databaseserver
+          permission: read-only
+    state: present
+
+# Base create user group with tag permissions
+- name: Create user group with tag permissions
+  community.zabbix.zabbix_usergroup:
+    server_url: "http://zabbix.example.com/zabbix/"
+    login_user: admin
+    login_password: secret
+    name: ACME
+    tag_filters:
+        - host_group: Webserver
+          tag: Application
+          value: Java
+        - host_group: Discovered hosts
+          tag: Service
+          value: JIRA
+    state: present
+
+# Base delete user groups example
+- name: Delete user groups
+  community.zabbix.zabbix_usergroup:
+    server_url: "http://zabbix.example.com/zabbix/"
+    login_user: admin
+    login_password: secret
+    name: ACME
+    state: absent
+'''
+
+RETURN = r'''
+state:
+  description: User group state at the end of execution.
+  returned: on success
+  type: str
+  sample: 'present'
+usergroup:
+  description: User group name.
+  returned: on success
+  type: str
+  sample: 'ACME'
+usrgrpid:
+    description: User group id, if created, changed or deleted.
+    returned: on success
+    type: str
+    sample: '42'
+msg:
+    description: The result of the operation
+    returned: always
+    type: str
+    sample: 'User group created: ACME, ID: 42'
+'''
+
+
+import traceback
+
+try:
+    from zabbix_api import ZabbixAPI
+    HAS_ZABBIX_API = True
+except ImportError:
+    ZBX_IMP_ERR = traceback.format_exc()
+    HAS_ZABBIX_API = False
+
+from ansible_collections.community.zabbix.plugins.module_utils.base import ZabbixBase
+import ansible_collections.community.zabbix.plugins.module_utils.helpers as zabbix_utils
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
```

Comment:
```suggestion
from ansible.module_utils.basic import AnsibleModule
```

otherwise it will fail sanity on unused import when you remove try/except for zabbix_api import